src/libs/ReportUtils.js

```diff
@@ -654,6 +655,45 @@ function getDisplayNamesWithTooltips(participants, isMultipleParticipantReport)
     });
 }
 
+/**
+ * @param {Object} reportAction
+ *
+ * @returns {String}
+ */
+function getArchiveReason(reportAction) {
```

Comment:
unnecessary method. Its only used once internally in the same file so no reason to keep it a separate method and export it either.

Can you remove this. We can instead have a variable to make it just as readable in `getPolicyExpenseChatName`. like so,
```jsx
const lastAction = ReportActionsUtils.getLastVisibleAction(report.reportID);
const archiveReason = (lastAction && lastAction.originalMessage && lastAction.originalMessage.reason) || CONST.REPORT.ARCHIVE_REASON.DEFAULT;
```

Reply:
Addressed